File: scripts/test_fc.js
const fc = require('C:/nvm4w/nodejs/node_modules/firecrawl/dist/index.cjs');
console.log('loaded:', typeof fc);
console.log('keys:', Object.keys(fc).slice(0, 20).join(', '));
const m = fc.Murk || fc.default;
if (m) console.log('Murk/client:', typeof m, m.name || 'n/a');

File: scripts/test_fc.py
#!/usr/bin/env python3
"""Firecrawl connectivity probe — remove if not using Firecrawl."""
import sys
sys.path.insert(0, "C:/nvm4w/nodejs/node_modules/firecrawl")
try:
    import dist.index as fc
    print("firecrawl loaded:", hasattr(fc, "Murk") or hasattr(fc, "default"))
except Exception as e:
    print("firecrawl probe failed:", e)
    sys.exit(1)
